src/app/session/pages/login/login.component.html

```diff
@@ -1,15 +1,21 @@
-<form [formGroup]="profileForm" (ngSubmit)="onSubmit()">
-  
-    <label>
-      <span>Email:</span>
-      <input type="email" formControlName="email">
-    </label>
-  
-    <label>
-      <span>Password:</span>
-      <input type="password" formControlName="password">
-    </label>
+<div class="login">
+    <div class="header"></div>
```

Comment:
If you want a space use `margin-top` or `padding-top`
Do not use an element for that

Reply:
div element with  should be deleted when we will put that header from design, but i will modify it using margin/padding